db/install.xml

```diff
@@ -128,10 +128,12 @@
         <FIELD NAME="questionid" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false"/>
         <FIELD NAME="state" TYPE="int" LENGTH="1" NOTNULL="true" DEFAULT="0" SEQUENCE="false" COMMENT="Property whether the question is approved, disapprove, new or changed."/>
         <FIELD NAME="hidden" TYPE="int" LENGTH="1" NOTNULL="true" DEFAULT="0" SEQUENCE="false" COMMENT="Property whether the question hidden or not."/>
+        <FIELD NAME="groupid" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="0" SEQUENCE="false" COMMENT="Group id"/>
```

Comment:
mod_forum uses -1 as the default value for its groupid column. Should we be consisten?

Reply:
I have updated it to -1 to make it be consistent with other activities:
https://github.com/studentquiz/moodle-mod_studentquiz/commit/de051efaf209b7514d41af7c0d8523b0d47f3d9d#diff-4013433adb48858dfeac13caf181742020b6c693bc853298312c45b134377cb6R132
https://github.com/studentquiz/moodle-mod_studentquiz/commit/de051efaf209b7514d41af7c0d8523b0d47f3d9d#diff-fcb317575e9e6245fdf1590e51a86685b61c2cc9573a9df8034323ccb3ef614fR834